website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,245 @@
+   - This removes the need to save passwords and secret environment variables locally.
+
+4. In **Account settings**, enable [partial parsing](/docs/deploy/deploy-environments#partial-parsing) to only reparse changed files, saving time.
+
+5. In **Account settings**, enable [Git repo caching](/docs/deploy/deploy-environments#git-repository-caching) for job reliability & third-party outage protection. <Lifecycle status='enterprise' />
+
+## Data platform setup
+
+This section outlines the considerations and methods to connect your data platform(s) to dbt Cloud.
+
+1. Set up your [data platform connections](/docs/cloud/connect-data-platform/about-connections) and [environment variables](/docs/build/environment-variables) in dbt Cloud. dbt Cloud can connect with a variety of data platform providers including:
+   - [AlloyDB](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) 
+   - [Amazon Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) 
+   - [Apache Spark](/docs/cloud/connect-data-platform/connect-apache-spark) 
+   - [Databricks](/docs/cloud/connect-data-platform/connect-databricks) 
+   - [Google BigQuery](/docs/cloud/connect-data-platform/connect-bigquery)
+   - [Microsoft Fabric](/docs/cloud/connect-data-platform/connect-microsoft-fabric)
+   - [PostgreSQL](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb)
+   - [Snowflake](/docs/cloud/connect-data-platform/connect-snowflake)
+   - [Starburst or Trino](/docs/cloud/connect-data-platform/connect-starburst-trino)
+
+2. You can verify your data platform connections by clicking the **Test connection** button in your deployment and development credentials settings.
+
+### Additional configuration
+- Use [OAuth connections](/docs/cloud/manage-access/set-up-snowflake-oauth), which enables secure authentication using your data platform’s SSO.  <Lifecycle status='enterprise' />
+
+## Git setup
+
+Your existing dbt project source code should live in a Git repository. In this section, you will connect your existing dbt project source code from Git to dbt Cloud.
+
+1. Ensure your dbt project is in a Git repository.
+
+2. In **Account settings**, select **Integrations**, and [connect and configure Git](/docs/cloud/git/git-configuration-in-dbt-cloud) in dbt Cloud to connect your Git repository:
+   - Connect with one of the [native integrations](/docs/cloud/git/git-configuration-in-dbt-cloud) in dbt Cloud (such as GitHub, GitLab, and Azure DevOps).
+   - [Import a Git repository](/docs/cloud/git/import-a-project-by-git-url) from any valid Git URL that points to a dbt project.
+
+### Additional configuration
+- Log into dbt Cloud using [OAuth connections](/docs/cloud/git/connect-github) to integrate with your source code platform. It automatically links to the repository using one of the native integrations set at the account level. <Lifecycle status='enterprise' />
```

Comment:
Does this need to be a bullet point? It's under a different header and it doesn't line up with the bullets under #2 if it's supposed to be a part of that I think we should remove the header and line it up, otherwise we can probably remove the bullet and make it a paragraph

Reply:
it's an optional config so it is a step for those that are enterprise. that's why i initially had it as a numbered bullet. i think removing the header hides it a little and i'd like to make this optional config more visible. maybe i can combine both and number it as a 1:

### Additional configuration
1. Log into dbt Cloud using [OAuth connections](/docs/cloud/git/connect-github) to integrate with your source code platform. It automatically links to the repository using one of the native integrations set at the account level. <Lifecycle status='enterprise' />